pkg/operator/hive/hive.go

```diff
@@ -4,22 +4,23 @@ import (
 	"bytes"
 	"context"
 	"crypto/md5"
+	"encoding/hex"
 	"fmt"
 	"os"
 	"strconv"
 	"strings"
 
 	log "github.com/sirupsen/logrus"
 
+	oappsv1 "github.com/openshift/api/apps/v1"
 	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
 	"github.com/openshift/hive/pkg/constants"
 	hiveconstants "github.com/openshift/hive/pkg/constants"
 	"github.com/openshift/hive/pkg/controller/images"
+	"github.com/openshift/hive/pkg/controller/utils"
 	"github.com/openshift/hive/pkg/operator/assets"
 	"github.com/openshift/hive/pkg/operator/util"
 	"github.com/openshift/hive/pkg/resource"
-
-	oappsv1 "github.com/openshift/api/apps/v1"
```

Comment:
Don't move this. The non-hive "github.com/openshift" imports should be in their own section. The hive imports should be last, but the imports in this file are out-of-order.